gdcdictionary/schemas/program.yaml

```diff
@@ -36,6 +36,7 @@ properties:
   name:
     type: string
     description: "Full name/title of the program."
+    pattern: "^[A-Z0-9]+|[0-9]+|[A-Z]+$"
```

Comment:
Hi Phillis, it is saying that we can have project names with numbers or uppercase characters only, it is hard to see but there are pipes (or's) in between. Probably there is a better why of structuring this. I am open to suggestions :)
>>> import re
>>> pattern = "^[A-Z0-9]+|[0-9]+|[A-Z]+$"
>>> re.match(pattern, "A")
<_sre.SRE_Match object at 0x7efea15d9370>
>>> re.match(pattern, "9")
<_sre.SRE_Match object at 0x7efea15d93d8>
>>> re.match(pattern, "999")
<_sre.SRE_Match object at 0x7efea15d9370>
>>> re.match(pattern, "AAAA")
<_sre.SRE_Match object at 0x7efea15d93d8>
>>> re.match(pattern, "9AAAA")
<_sre.SRE_Match object at 0x7efea15d9370>
>>> re.match(pattern, "A9AAAA")
<_sre.SRE_Match object at 0x7efea15d93d8>

However I just found another bug, thanks for asking!